### DISCUSSION TITLE
Installing to wrong drive on Win10

### DISCUSSION THREAD

**Discussion by ryderse** [rapid7/metasploit-framework#15272] [General] [answered]
I just installed the latest version (v6.0.46) on my win10 PC. I have an SSD for the c: drive so I want to install this on my d: drive instead. I clearly selected the d: drive but for some reason it keeps getting installed on g: instead. Why would that be? 

I'm going to disconnect my g: drive and try again but thought I'd mention this. I'm not sure if this is the right place for this discussion, apologies if so. Please point me to the right place if so. 

Thank you
-Steve

Update: I disconnected the g: drive, re-installed to d: The installed looked like it worked, it added the folder on d: to my path. However, after the install, i see the install folder is empty. :-/. Looks like this insists on being installed on c: drive?

**Comment by bcoles** [comment, marked as answer]
Issues with the installer are tracked on the https://github.com/rapid7/metasploit-omnibus/ repository.

This is a bug. This issue is tracked here: https://github.com/rapid7/metasploit-omnibus/issues/115
